### PR TITLE
V24A-608 Database out of memory

### PR DESCRIPTION
**Pull request by Wouter99**
Database opening went incorrectly. Furthermore database closes now in defer.

### CODE REVIEW

File: backend/gamification/gamification.go [hunk 1]
```diff
@@ -56,6 +55,15 @@ func PointCalculation(gs GameState, scanResults []checks.Check) (GameState, erro
 		}
 		gs.PointsHistory = append(gs.PointsHistory, gs.Points)
 	}
+	// Close the database
+	logger.Log.Info("Closing database")
```
Comment (olafboekholt) [inline backend/gamification/gamification.go:59]: I would make it a debug log, to keep info logging as clean as possible

File: backend/tray/tray.go [hunk 2]
```diff
@@ -535,6 +538,7 @@ func changeNextScan(settings usersettings.UserSettings, value int) {
 // Parameters:
 //   - scanInterval int: The scan interval in hours.
 //
+// t
```
Comment (olafboekholt) [inline backend/tray/tray.go:541]: why is this t added?

**Review by olafboekholt — CHANGES_REQUESTED** [review]
Make a Jira bug and rename the branch and pull request with the Jira bug number (the V24A...), to help keep track of this pr.

File: backend/gamification/gamification.go (hunk 1 above)
Comment (Wouter99) [reply backend/gamification/gamification.go]: I agree. It was however also in other parts of the code, so lets in the future go over all logging for consistency

**Review by olafboekholt — APPROVED** [review]

File: backend/gamification/gamification.go (hunk 1 above)
Comment (olafboekholt) [reply backend/gamification/gamification.go]: yes that is something that needs to be done, good point

**Pull request merged**